pkg/collector/corechecks/servicediscovery/module/impl_linux_test.go

```diff
@@ -503,6 +504,21 @@ func assertStat(t assert.TestingT, svc model.Service) {
 	assert.Equal(t, uint64(createTimeMs/1000), svc.StartTimeSecs)
 }
 
+func assertCPU(t *testing.T, url string, pid int) {
+	proc, err := process.NewProcess(int32(pid))
+	require.NoError(t, err, "could not create gopsutil process handle")
+
+	// Compare CPU usage measurement over an interval.
+	_ = getServicesMap(t, url)
+	referenceValue, err := proc.Percent(1 * time.Second)
+	require.NoError(t, err, "could not get gopsutil cpu usage value")
+
+	// Calling getServicesMap a second time us the CPU usage percentage since the last call, which should be close to gopsutil value.
+	portMap := getServicesMap(t, url)
+	assert.Contains(t, portMap, pid)
+	assert.InDelta(t, referenceValue, portMap[pid].CPUUsage, 0.10)
```

Comment:
Maybe `InEpsilon` would be better since it is less dependent on the absolute value?

Reply:
The problem is our test services are currently not really doing anything. The value that gopsutil reports has a high chance of being 0, which `InEpsilon` does not accept as an expected value.